bin/env2options

```diff
@@ -78,17 +89,20 @@ def parse_args():
 def main():
     args = parse_args()
     if args.source == SOURCE_ALL:
-        values = get_merged_settings()
+        job_cmd = get_job_cmd()
+        merged_settings = get_merged_settings()
```

Comment:
I am not sure we are reading the same code for get_merged_settings.

My understanding is that settings are merged in the order defined by https://github.com/scrapinghub/scrapinghub-image-casperjs/blob/3dfa56db3eebeebfc1397b242caf6d80b99dc264/bin/env2options#L17-L22